### PR TITLE
[CARBONDATA-2138] Added documentation for HEADER option while loading data

### PR DESCRIPTION
**Pull request by sgururajshetty**
Added documentation for HEADER option in load data

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

**Comment by ravipesala** [comment]
Can one of the admins verify this patch?

File: docs/data-management-on-carbondata.md [hunk 1]
```diff
@@ -330,6 +330,14 @@ This tutorial is going to introduce all commands and data operations on CarbonDa
...
 
+  - **HEADER:** When you load the CSV file without the file header and the file header is the same with the table schema, then add 'HEADER'='false' to load data SQL as user need not provide the file header. By default the value is 'true'.
+  false: CSV file is without file header.
+  true: CSV file is with file header.
+  
+    ```
+    OPTIONS('HEADER'='false') 
+    ```
+
   - **FILEHEADER:** Headers can be provided in the LOAD DATA command if headers are missing in the source files.
```
Comment (QiangCai) [inline docs/data-management-on-carbondata.md]: suggest mentioning: If HEADER option exists and the value is true, not require FILEHEADER option.

**Comment by sgururajshetty** [comment]
@QiangCai fixed the review comment. Kindly review and merge.

**Comment by jackylk** [comment]
LGTM

**Pull request closed without merging**